core/src/main/java/com/ultramixer/jarbundler/AppBundleProperties.java

```diff
@@ -25,6 +25,8 @@
 // import java.util.Scanner;
 import java.util.List;
 
+// Java language imports
+import java.lang.String;
```

Comment:
no need to import `java.lang.*` classes here.

Reply:
No problem.